### DISCUSSION TITLE
Difference between IntelliJ  and VSCode

### DISCUSSION THREAD

**Discussion by SumnanAzadi** [one-dark/vscode-one-dark-theme#48] [Q&A] [answered]
One dark theme is my favorite theme for any IDE by IntelliJ. But I noticed there is a significant difference between VSCode and Intellij version, It's really noticeable especially when I work on python. Is there any way to export the color scheme from IntelliJ and import it to VSCode?

**Comment by mskelton** [comment, marked as answer]
IntelliJ and VS Code use entirely different theming systems, so there isn't a great way to just export one to the other.  I don't do much with either theme anymore, as @unthrottled maintains the JetBrains theme (big props to him for doing that, he's done an awesome job maintaining it) and I no longer use VS Code myself so I don't put much into this theme anymore.  If you are interesting in contributing, I'd be happy to walk you through what the process of contributing changes to this theme looks like.

**Reply by SumnanAzadi** [reply]
Thanks mark for the reply. I actually went through the theming mechanism in VS code yesterday. Tweaked a little bit via `tokenColorCustomizations`, But sure, If you have any source materials you can share with me regarding this contribution to this theme, that would be great. BTW, what theme do you use these days?

**Reply by mskelton** [reply]
I've got some [docs here](https://github.com/one-dark/vscode-one-dark-theme/blob/main/CONTRIBUTING.md) for how to contribute, gives an overview of how to develop the theme as well as outlines the color palette.

As for what theme I use these days, I use [Tokyonight](https://github.com/folke/tokyonight.nvim) in Neovim.  Still a big fan of One Dark, just got tired of maintaining a theme in yet another editor since I'm not a great fan of the Neovim One Dark theme as the colors are not quite to my liking compared to my VS Code version.
